src/libsyntax/ast.rs

```diff
@@ -456,7 +472,7 @@ pub enum MetaItemKind {
     /// List meta item.
     ///
     /// E.g. `derive(..)` as in `#[derive(..)]`
-    List(InternedString, Vec<P<MetaItem>>),
+    List(InternedString, Vec<MetaListItem>),
```

Comment:
Yeah, sorry, that is what I meant (though iirc meta-items do end up getting cloned here and there).